docs/rules/no-unsafe-finally.md

```diff
@@ -61,7 +61,7 @@ JavaScript suspends the control flow statements of `try` and `catch` blocks unti
 
 ## Rule Details
 
-This rule disallows `return`, `throw`, `break`, and `continue` statements inside `finally` blocks. It allows indirect usages, such as in `function` or `class` definitions.
+This rule disallows `return`, `throw`, `yield`, `break`, and `continue` statements inside `finally` blocks. It allows indirect usages, such as in `function` or `class` definitions.
```

Comment:
Maybe "`return`, `throw`, `break`, `continue` statements and `yield` expressions" is better

Reply:
It seems `yield *` is missed.

Reply:
Thanks!